docs/usage/index.md

```diff
@@ -42,6 +42,33 @@ To avoid confusion, note that:
  - `cdn.jsdelivr.net/npm/pyodide@0.18.0/` is a mirror of the Pyodide NPM package, which includes none of the WASM files
 ```
 
+### Supported browsers
+
+Pyodide works in any modern web browser with WebAssembly support.
+
+**Tier 1** browsers are tested as part of the test suite with continuous integration,
```

Comment:
The wording is largely inspired by https://doc.rust-lang.org/nightly/rustc/platform-support.html